src/Build/BackEnd/Components/RequestBuilder/Lookup.cs

```diff
@@ -1411,6 +1412,31 @@ internal Scope(Lookup lookup, string description, IItemDictionary<ProjectItemIns
                 _propertySets = null;
                 _threadIdThatEnteredScope = Environment.CurrentManagedThreadId;
                 _truncateLookupsAtThisScope = false;
+                Next = lookup._lookupScopes;
+            }
+
+            /// <summary>
+            /// The next scope in the stack, if any.
+            /// </summary>
+            internal Scope Next { get; }
```

Comment:
Should this be called `Parent`? That's what the scope-to-scope/lookup-to-lookup relationship, right?